lib/MiLight/RadioStack.h

```diff
@@ -2,33 +2,55 @@
 #include <PL1167_nRF24.h>
 #include <MiLightRadioConfig.h>
 #include <MiLightRadio.h>
+#include <MiLightRadioPL1167_LT8900.h>
+#include <MiLightRadioInterface.h>
+#include <Settings.h>
 
 #ifndef _RADIO_STACK_H
-#define _RADIO_STACK_H 
+#define _RADIO_STACK_H
 
 class RadioStack {
```

Comment:
Good idea - the RF24 instance actually irritated me there, but again, I didn't wanted to mess too much around with your code.